docs/source/messages.rst

```diff
@@ -53,10 +53,4 @@ You can also log tracebacks when your code hits an unexpected exception:
             try:
                  dosomething()
             except:
-                 write_traceback(self.logger, u"yourapp:yourclass")
-
-The final argument to ``write_traceback`` is the "system".
-This should be a Unicode string, a logical description of what subsystem in your application generated the message.
-Colons are used as namespace separators by convention to discourage the use of Python modules and namespaces.
-System strings should involve code structure rather than file structure.
-This means they will not have to change if you decide to refactor your implementation.
+                 write_traceback(self.logger)
```

Comment:
I think this might be a good place to mention that `write_traceback` (and `writeFailure`) should use the context of the action to describe where the traceback originated.